utils/js/LogUtils.ts

```diff
@@ -103,7 +121,51 @@ function parseSource(source) {
     return '';
 }
 
-function writeLog(msg, source, level) {
-    if (source) msg = `[${source}] ${msg}`;
-    level === 'info' ? console.log(msg) : console.debug(msg);
+function writeLog(msgs: Some<any>, source, level: LogLevel) {
```

Comment:
this is alway an array, fwiw

Reply:
Updated to `msgs: Array<any>` and removed castArray